master/internal/api_experiment.go

```diff
@@ -43,3 +53,104 @@ func (a *apiServer) GetExperiments(
 	a.sort(resp.Experiments, req.OrderBy, req.SortBy, apiv1.GetExperimentsRequest_SORT_BY_ID)
 	return resp, a.paginate(&resp.Pagination, &resp.Experiments, req.Offset, req.Limit)
 }
+
+func (a *apiServer) ActivateExperiment(
+	ctx context.Context, req *apiv1.ActivateExperimentRequest) (
```

Comment:
non-blocking: this indentation is quite confusing. Take a look at some of the other endpoints.

Reply:
couldn't find an example in api_agents, users, or models can you share what you think looks better?